trio/_core/_tests/test_multierror.py

```diff
@@ -391,7 +398,7 @@ def simple_filter(exc):
         gc.garbage.clear()
 
 
-def assert_match_in_seq(pattern_list, string):
+def assert_match_in_seq(pattern_list: List[str], string: str) -> None:
```

Comment:
Why old generic if we are using `__future__` annotations?

Reply:
uh, no reason at all. Might've been something `pyannotate` did

Reply:
should also get autofixed by ruff pyupgrade